test/index.js

```diff
@@ -34,8 +36,9 @@ describe('Lob', function () {
     var options = { host: 'http://test' };
     var Lob     = require('../lib')(API_KEY, options);
 
-    Lob.countries.list()
-    .catch(function (err) {
+    Lob.addresses.list()
+    .catch(err => err)
+    .then(err => {
```

Comment:
why did you add a `.then` after the `catch` instead of putting the `expect` in the `.catch`

Reply:
This was just a [habitual thing](https://www.notion.so/lob/Testing-634ae35736654c84a205ca06acc10fa6#f3f8e64246ea4da58510769e15b7f770), I'm not sure its actually necessary.